reporter.go

```diff
@@ -8,59 +8,34 @@ import (
 	"time"
 
 	"github.com/aws/aws-lambda-go/lambdacontext"
-	"github.com/rcrowley/go-metrics"
-	"github.com/wavefronthq/go-metrics-wavefront"
 )
 
-// incrementCounter increments the counter by the given value if report is true
-func incrementCounter(counter metrics.Counter, value int64, report bool) {
+// updateCounter increments the value of a given counter if report is true
+func updateCounter(counter *float64, inc int64, report bool) {
```

Comment:
nit: should we name this as updateMetric instead ? 
this seems to be used to update all types of metrics counters, gauges etc.

Reply:
I don't think that's _nit_, I think that makes total sense 😄